calcom/cal.com
# feat: monthly email digest

## What does this PR do?



Fixes #10026
/claim #10026

https://github.com/calcom/cal.com/assets/47187878/95fb4e19-2463-4b6b-adca-cb4199d5cb6d

<img width="818" alt="Screenshot 2023-08-06 at 10 19 09 PM" src="https://github.com/calcom/cal.com/assets/47187878/312d6e2e-e9ca-4d78-b5d5-66b1a793643f">

## Technical Details

- Have used a cron job which will run every day via the GitHub workflows
- The cron job will check for which teams have completed another month today and will send an email if it has the required amount of data mainly if there were bookings for this team this month.

## Type of change


## How should this be tested?
- Can test this email by visiting the link localhost:3000/api/cron/monthlyDigestEmail on your local setup
- Make sure that you have teams that have completed 30 days, and you'll receive the email only if you are the owner of the team
- For email testing purposes can workaround it by removing the 30 days condition in apps/web/pages/api/cron/monthlyDigestEmail

## Mandatory Tasks
